### PR TITLE
Implement API token instructions

### LINKED ISSUE #14: API token instructions
Related to #3, but more urgent. I would include a link to this page in the authentication dialog of my plugin.

### PR DESCRIPTION
**Pull request by thepieterdc**
Instructions on how to generate an API token. Only in English for now.

Fixes #14

### CODE REVIEW

File: guides/creating-an-api-token/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Creating an API token
+description: "Tutorial: creating an API token"
+permalink: /guides/creating-an-api-token/
+parent: Handleidingen
+nav_order: 99
```
Comment (bmesuere) [inline guides/creating-an-api-token/index.md]:
```suggestion
nav_order: 98
```

File: guides/creating-an-api-token/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+lang: en
+---
+
+# Creating an API token
+
+By creating an API token, you can authorize external applications, such as the [Dodona plugin for JetBrains IDEs](https://plugins.jetbrains.com/plugin/11166-dodona), to act on your behalf, without ever providing them with your password. To create an API token, simply follow these five steps:
+
+## 1. Profile page
```
Comment (bmesuere) [inline guides/creating-an-api-token/index.md]: add verb to title to be in line with others

File: guides/creating-an-api-token/index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+First, go to your profile page by clicking the link in the dropdown menu.
+![My Profile](my-profile.png)
+
+## 2. Edit your profile
+At your profile page, click the yellow `EDIT`-button at the bottom.
+![Edit your profile](edit.png)
```
Comment (bmesuere) [inline guides/creating-an-api-token/index.md]: create a screenshot with more context instead of just the button

**Comment by thepieterdc** [comment]
I have implemented all of your remarks @bmesuere .

**Pull request merged**